src/client/components/DateTimePicker/DateTimePicker.styles.css

```diff
@@ -0,0 +1,23 @@
+.date-time-picker,
+input {
+  position: relative;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+  background: #f6f6f6;
+  border: 1px solid #e8e8e8;
+  box-sizing: border-box;
+  height: 50px;
+  width: 289px;
+  border-radius: 100px;
+  text-align: center;
+}
+
+/* input {
```

Comment:
Removing commented codes could be good practice.  Looks great !!

Reply:
That's true, removing commented codes is a good practice. I have done the same thing and was guided to keep the commented code locally, while the clean code is committed